bin/add_ext_reference.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+
+import requests
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+from ebi_eva_internal_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_internal_pyutils.pg_utils import get_all_results_for_query, execute_query
+from retry import retry
+
+from eva_submission.eload_utils import check_project_exists_in_evapro, check_existing_project_in_ena
+from eva_submission.submission_config import load_config
+
+logger = log_cfg.get_logger(__name__)
+
+
+def add_external_reference_to_ena(project_accession, source_database, identifier):
```

Comment:
Maybe log that this isn't supported yet (assuming we'll use the script in production before this is implemented, if you're planning on implementing it soon then ignore this).